### PR TITLE
Johnmckellar patch 1

### PR DESCRIPTION
**Pull request by johnmckellar**
@KellyVance @BlakePaul @predam1 

This vocab was withdrawn as it was deemed stale for want of review.

A new review is requested in terms of the status of a geologic feature. Please compare with GA's ASUD and consider what is required in terms of our needs for RU-data transfer from MERLIN.

Micaela, I have thus added you to the review list.

Cheers,

John

### CODE REVIEW

**Review by KellyVance — CHANGES_REQUESTED** [review]
Generally need to upscale this to make it more general and widely applicable.
Fairly straight forward changes

File: vocabularies/gsq-feature-status.ttl [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    skos:prefLabel "Reserved by Geoscience Australia"@en .
+
+fst:UnnamedRockUnit a skos:Concept ;
+    skos:broader fst:InformalRockUnit ;
+    skos:definition "The status of a rock unit that has been recorded in published or unpublished reports, records, maps, etc., but has not been named."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/gsq-feature-status> ;
+    skos:notation "fstun" ;
+    skos:scopeNote "Geoscience Australia definition supported here, additional to above: A distinctive area of rocks (polygon(s) in a dataset), where the rocks have been assigned a description, but not a name. Often used for subsurface units that are based on geophysical properties alone. Also used for very broad-scale maps where named units are too small to be displayed at the intended scale. ASUD does not have the resources to record all of these unnamed units created by all the State and Territory Surveys in any useful way, but this organisation’s current GIS dataset structure requires a 'stratno' for all geological polygons, so it does currently record any unnamed units used in its datasets."@en ;
+    skos:prefLabel "Unnamed rock unit"@en .
```
[the remark this answers is not in the document]
Comment (johnmckellar) [reply vocabularies/gsq-feature-status.ttl:94]: Vance (& Paul), just another note on DEF (now formal): Not all RUs meet the definition provided by Paul (which was derived from MERLIN?) in that some are informal. This largely encompasses RUs defined as 'beds', which are not recognized as formal units. These will need to be weeded out.

**Comment by johnmckellar** [comment]
I take it that I need to close this!!??

File: vocabularies/gsq-feature-status.ttl (hunk 1 above)
[the remark this answers is not in the document]
Comment (KellyVance) [reply vocabularies/gsq-feature-status.ttl:95]: @johnmckellar 
If you agree with the unnamed edit, can you please commit that, and if not update as needed?
As part of the attribute mapping we can include the IF status = DEF and strat = beds -> Informal

Once those commits are done i'll merge and close the pull request. Then i'll be able to update our vocprez

**Review by KellyVance — APPROVED** [review]

**Pull request merged**